docs/getting-started/steps-to-build-deploy-contract.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Steps to build and deploy a smart contract
+sidebar_position: 8
+---
+
+This is an explanation of the process of building a smart contract in Marlowe from start to deployment on the preview or pre-production network of the Cardano blockchain. 
+
```

Comment:
Would be great to add prerequisites. Please talk to SMEs to define and add those.

Reply:
Edit: since the page breaks down content into explainers and guidelines, it would be better to add prerequisites below in the development part.

Reply:
Good point about prerequisites. I will ask for more input on this topic. I recommend having a single section for prerequisites near the top of the page that covers all the essential aspects of caution for both contract building and deployment. I'll start with this: 

## Prerequisites

* See [**Learning Marlowe**](https://docs.marlowe.iohk.io/docs/getting-started/learning-marlowe) to familiarize yourself with the tutorials and other recommended topics. 
* See the tutorial about [**checking the safety of a Marlowe contract**](https://github.com/input-output-hk/marlowe-starter-kit/blob/main/lessons/07-safety/07-safety.ipynb)
* See also [**Using Marlowe safely**](https://github.com/input-output-hk/marlowe-starter-kit/blob/main/docs/using-marlowe-safely.md)
* [**Considerations before you deploy**](https://docs.marlowe.iohk.io/docs/getting-started/runner#considerations-before-you-deploy)